### ISSUE TITLE
Error running project locally - cannot use stream

### ISSUE THREAD

**Issue by PylotLight** [charmbracelet/mods#269]
**Describe the bug**
Can't run locally, error:
cannot use stream (variable of type *"github.com/sashabaranov/go-openai".ChatCompletionStream) as chatCompletionReceiver value in struct literal: *"github.com/sashabaranov/go-openai".ChatCompletionStream does not implement chatCompletionReceiver (wrong type for method Close)
		have Close() error
		want Close()compilerInvalidIfaceAssign
var stream *openai.ChatCompletionStream

**Setup**
Please complete the following information along with version numbers, if applicable.
 - OS WSL
 - Shell bash

*Note: you might encounter rendering issues if your locale does not use
`UTF-8` encoding. Please check your locale (`locale` on POSIX systems) to
see what encoding is being used by your system.*

**To Reproduce**
git clone [repo]
cd mods
go run .

**Source Code**
errors coming from stream.go and mods.go

**Expected behavior**
Mods program runs successfully

**Screenshots**

**Additional context**

**Comment by PylotLight** [comment]
I got it working after updating the package version for open-ai sdk and the return values. 
Will submit PR for review of this one but wanted issue created first.

**Comment by caarlos0** [comment]
hey!

how did you installed mods? I don't see how you would get that by using the current recommended installation methods, as it looks like some dependency was updated, which is what's causing the problems...

**Comment by PylotLight** [comment]
> git clone [repo]
> cd mods
> go run .

I already added the steps I used in OP. 
Also correct some dependencies were updated which I fixed in the attached PR.

**Comment by caarlos0** [comment]
the steps in the OP would not have updated the deps... so I would advise to `git checkout -- .`

**Comment by PylotLight** [comment]
> the steps in the OP would not have updated the deps... so I would advise to `git checkout -- .`

Right, it was broken as is. I was getting that error with no changes. 
There was nothing to revert so `git checkout -- .` achieves nothing.
